src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "CEA0FDFC6D3B4085823D60DC76F28100",
+  "ActionKeyword": "*",
+  "Name": "VSCode Workspaces",
+  "Description": "Opened VSCode Workspaces",
+  "Author": "ricardosantos9521",
+  "Version": "1.0.0",
+  "Language": "csharp",
+  "Website": "",
+  "ExecuteFileName": "Community.PowerToys.Run.Plugin.VSCodeWorkspaces.dll",
+  "IcoPath": "Images\\code.png"
```

Comment:
I can't find the img to look at it. Want to know how it looks like. Maybe it's layout should aligned with the other plugin icons.

Reply:
This icon will be used in the settings in the future.

Reply:
well caught